flang/test/Lower/PowerPC/ppc-vec-shift-be-le.f90

```diff
@@ -1,4 +1,4 @@
-! RUN: %flang_fc1 -flang-experimental-hlfir -emit-llvm %s -triple ppc64le-unknown-linux -o - | FileCheck --check-prefixes="CHECK" %s
+! RUN: %flang_fc1 -flang-experimental-hlfir -emit-llvm %s -triple ppc64le-unknown-linux -o - | FileCheck --check-prefixes="LLVMIR" %s
```

Comment:
Isn't this makes `CHECK` prefixed unused? For example, on 10 line.

Reply:
Yes, you're right. I will update it. Thanks.

Reply:
While currently `CHECK-LABEL` is allowed check prefix, this soon can be not (https://github.com/llvm/llvm-project/pull/92735). Plus this converts https://llvm.org/docs/CommandGuide/FileCheck.html#the-check-label-directive to simple prefix(or not? Linked pr should fix that).

Reply:
Thanks for the heads up. I think I'd better use another prefix.

Reply:
I'm losing confidence in my understanding how different prefixes works between them: in case of
```
FOO-LABEL: xxx
BAR: yyy
...
FOO2-LABEL: zzz
```
will BAR me matched between FOO and FOO2 or not? idk.

Anyway, in https://github.com/llvm/llvm-project/pull/92943/commits/76100a9265b41176ca2e89d3668d8915426bfdcf you defined prefixes ending with `-LABEL`, but they should be `LLVM`, i guess. So you need to change only definitions.

Reply:
> Anyway, in [76100a9](https://github.com/llvm/llvm-project/commit/76100a9265b41176ca2e89d3668d8915426bfdcf) you defined prefixes ending with `-LABEL`, but they should be `LLVM`, i guess. So you need to change only definitions

Sorry, I don't quite understand. Did you mean if I specify `LLVM` in `--check-prefixes`, `LLVM-LABEL` will be effective?

Reply:
Yes, this should start working as directive `LABEL`, if i understand correct.